cli/internal/fs/turbo_json.go

```diff
@@ -161,7 +203,10 @@ func LoadTurboConfig(rootPath turbopath.AbsoluteSystemPath, rootPackageJSON *Pac
 	for scriptName := range rootPackageJSON.Scripts {
 		if !turboJSON.Pipeline.HasTask(scriptName) {
 			taskName := util.RootTaskID(scriptName)
-			turboJSON.Pipeline[taskName] = TaskDefinition{}
+			turboJSON.Pipeline[taskName] = BookkeepingTaskDefinition{
+				fieldsMeta:     map[string]bool{},
+				TaskDefinition: TaskDefinition{},
+			}
```

Comment:
Reading through the code, I don't think we need to initialize `fieldsMeta` and can just use the zero value of this struct. Only place we write to `fieldsMeta` is in `UnmarshalJSON` which initializes it right away.
```suggestion
			turboJSON.Pipeline[taskName] = BookkeepingTaskDefinition{}
```

Reply:
done, good suggestion